docs/UserGuide.md

```diff
@@ -1,79 +1,69 @@
 ---
   layout: default.md
   title: "User Guide"
-  pageNav: 3
 ---
-
-![wardwatch-logo](images/wardwatch_banner.png)
```

Comment:
was it intentional to remove the entire image?

Reply:
Nope, good spot. I have changed it.